### ISSUE TITLE
website_rank 

### ISSUE THREAD

**Issue by Rockyzsu** [lijiejie/website_rank#1]
Thanks your nice PR value check python script.

I have a question here: 
after inserting some info into you script ( in function get_rank()   
print url 

output below link 
http://pr.chinaz.com//ajaxsync.aspx?at=pr&enkey=ACmVoHVYfWigbzsUtNaWlzVbkEzbTVPK&url=www.qq.com

http://pr.chinaz.com//ajaxsync.aspx?at=sougoupr&enkey=ACmVoHVYfWigbzsUtNaWlzVbkEzbTVPK&url=www.qq.com

But when i try to paste above link into the browser, it can't not get the right feedback, it is empty page.

So why this can get result from python script ? IS it caused by cookiejar ?

**Comment by lijiejie** [comment]
That's right. pr.chinaz.com will check weather enkey value can match your cookie. cookiejar creates a cookie manager for urllib2.
